app/controllers/users_controller.rb

```diff
@@ -104,14 +104,19 @@ def remove_association
   end
 
   def onboarding_update
-    current_user.assign_attributes(params[:user].permit(:summary, :location, :employment_title, :employer_name, :last_onboarding_page)) if params[:user]
+    if params[:user]
+      current_user.assign_attributes(params[:user].permit(:summary, :location, :employment_title, :employer_name, :last_onboarding_page))
```

Comment:
this still looks too long on my screen, is at column 138, maybe like this:

```suggestion
      user_params = %i[summary location employment_title employer_name last_onboarding_page]
      current_user.assign_attributes(params[:user].permit(user_params))
```

Reply:
I actually wanted to break up those lines, but I knew I would have to fight Rubocop to do them the way I wanted to, and thought that others might not share my preference to shorten those lines, so I refrained.

I've just done it, and followed your suggestions...except that in %?[] expressions I like to separate words by 2 spaces; I find it's way easier to see the boundaries. Is that ok?

My preference is for short lines. I travel for months at a time and when I do have only my 13" Macbook Pro display; and I usually have a directory tree pane on the left side, leaving even less room for code. However, I realize that most developers routinely use large screens, and I want to accommodate them too.

Do we have some kind of guidance as to general maximum line length, so that I can know when to indulge my preference, and when to subjugate it?

By the way, for many years I have been using the convention of 2 indentation levels for line continuations; I've found it to be the optimal solution, as opposed to:

* continuing below the opening paren of the previous line, which can waste dozens of spaces, and
* continuing only one indentation level to the right, which fails to differentiate between a line continuation and a logical nesting level change

Unfortunately, most Rubyists I've met are not aware that this convention exists, and the usual Rubocop configurations don't permit this.

Reply:
> Do we have some kind of guidance as to general maximum line length, so that I can know when to indulge my preference, and when to subjugate it?

We do, it's 100, but it's not actually being respected that much since the configuration is disabled in https://github.com/thepracticaldev/dev.to/blob/master/.rubocop_todo.yml

At some point we'll have to enforce it but I think the best way is to get there one step at a time